JamfSync/Model/Jcds2Dp.swift

```diff
@@ -200,33 +207,453 @@ class Jcds2Dp: DistributionPoint {
         } else {
             fileUrl = file.fileUrl
         }
+        
         guard let fileUrl else { throw DistributionPointError.badFileUrl }
+        
```

Comment:
Get rid of the second blank line. If we ever add SwiftLint, it will whine about it.